backend/src/apiserver/common/const.go

```diff
@@ -62,10 +62,14 @@ const (
 )
 
 const (
-	GoogleIAPUserIdentityHeader string = "x-goog-authenticated-user-email"
-	GoogleIAPUserIdentityPrefix string = "accounts.google.com:"
+	GoogleIAPUserIdentityHeader    string = "x-goog-authenticated-user-email"
+	GoogleIAPUserIdentityPrefix    string = "accounts.google.com:"
+	AuthorizationBearerTokenHeader string = "Authorization"
+	AuthorizationBearerTokenPrefix string = "Bearer "
 )
 
+const TokenReviewAudience string = "ml-pipeline"
```

Comment:
We're good with this yep. I'll push a commit to change it.

Ideally the audience needs to be as unique as possible. This among with other potential security reasons is why it is common to use the name of the Kubernetes service backing the deployment.
However this may make more difficult the configuration of both the deployment of KFP and clients communicating to it.

Also, how about making it configurable via an environment variable (ConfigMap essentially) and have `pipelines.kubeflow.org` as a sane default?

Reply:
Makes sense to me making it configurable with good default